### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by IraSoro** [IraSoro/peri#288] [General]
### General Discussion

Welcome to our general discussion! Here you can ask any questions, share feedback, ideas or just chat about our app. We are open to any topics!

#### Additional info

- **Bugs and Suggestions**: If you’ve found a bug or have ideas for enhancing the application, you can create a new discussion in the “Bugs and Issues” section or use the template to create an Issue. [Go to the "Bugs and Issues" section](https://github.com/IraSoro/peri/discussions/categories/bugs-and-issues).
- **Contribution to our project**: We welcome everyone who wants to participate in the development of the project! Here is a [link](https://github.com/IraSoro/peri/discussions/categories/q-a) to the discussions, where it is written in more detail.

#### Topics Open for Discussion:

- Feel free to write any questions, comments, or suggestions you may have here.
- Discuss new ideas or improvements you would like to see in the application.
- Share your thoughts and suggestions!

Your feedback is important to us, and we appreciate your contributions to the development of the project!

**Comment by julienmas** [comment]
Hi !
I have questions about user data. Are they used and stored locally, or are they sent at some point to a server ? Is their any kind of encryption to protect the data ?
Thanks a lot,
Julien

**Reply by imblowfish** [reply]
Hello, we don't send any user data to the server and it stored locally on your device

**Comment by Terib120** [comment]
Hello!
![Screenshot_2026-03-17-10-51-58-430_period tracker](https://github.com/user-attachments/assets/441ca378-cebd-4e8a-9c2b-f0e085608491)

"Period is possible today." It shows this somehow and I am not able to fix it. Also sometimes the screen just freezes. 

Although I really liked the app. The ui is simple and easy to navigate.
I wish there was a partner connect feature though, would really add to the app! :)

**Reply by imblowfish** [reply]
Hello, this is the first cycle, on your calendar you see that it should be started on March 8, but you didn't marked it as started with click on the "Mark" button. If you'll click on period start it'll show "Period 1st day" or any other status message related to cycle.

Thank you for your interest in our app, about partner connect feature, stay in touch, right now we working on the next peri 3.0 version :)
